alerting/src/main/kotlin/org/opensearch/alerting/alerts/AlertIndices.kt

```diff
@@ -88,11 +103,20 @@ class AlertIndices(
         /** The index name pattern referring to all alert history indices */
         const val HISTORY_ALL = ".opendistro-alerting-alert-history*"
 
+        /** The index name pattern referring to all alert history indices */
+        const val FINDING_ALL = ".opendistro-alerting-alert-finding-history*"
```

Comment:
Change the pattern to `".opendistro-alerting-finding-history*"`. Change the name to `FINDING_HISTORY_ALL`. Plus change the comment to reflect its for finding and not alerts